src/models/Offer.js

```diff
@@ -110,9 +110,28 @@ OfferSchema.query.current = function() {
     });
 };
 
+OfferSchema.query.activeOffersCount = function(canditateCompany) {
+    return this.where({
+        publishDate: {
+            $lte: new Date(Date.now()),
+        },
+        publishEndDate: {
+            $gt: new Date(Date.now()),
+        },
+        owner: {
+            $eq: canditateCompany.owner,
+        },
+    });
```

Comment:
If you look closely, this has the same first two conditions as Offer.current query, thus you can chain them and have a simpler one here: 

```js
OfferSchema.query.activeOffers = function(owner) {
    return this.current.where({owner});
}
```

Reply:
After looking at this, I noticed that the current query does not have the `if !isHidden`, should we add it? @miguelpduarte @GuilhermeJSilva? If we do, I'd change from `current` to `currentActive` or something like that, to make it clear that we are not only checking the dates

Reply:
I do not recall what the idea behind `isHidden` was anymore... Do any of you remember? I'm thinking if we should scrap it altogether if it's not being used... Since this is mongo we can always just introduce it later anyway 😛

Reply:
I am not sure, but when I was looking, I felt the need for a isActive, in case the company wants to hide/remove an offer, probably the isHidden is that field, that's why I suggested to use it in the `current` query